io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "vimix",
+    "rename-desktop-file": "vimix.desktop",
+    "rename-icon": "vimix",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--device=shm",
+        "--device=all",
```

Comment:
why do you need device all?

Reply:
i'm open to learn: the `all` option was suggested at several places in the documentation.

vimix should be able to access cameras (ie. webcams with v4l2), microphone (gstreamer audio visualization), joysicks, tablet (e.g. wacom) and obviously keyboard and mouse.